### PR TITLE
QA cookie banner rule updates

### PR DESCRIPTION
**Pull request by Softvision-PatriciuPop**
Adding 74 new rules for:
 salesforce.com, discord.com, reuters.com, booking.com, wsj.com, bloomberg.com, wix.com, businessinsider.com, 20min.ch, google.at, google.be, google.de, orf.at, abv.bg, dr.dk, google.it, google.pt, aktuality.sk, google.bg, jutarnji.hr, iltalehti.fi, orange.fr, google.gr, google.hu, google.nl, onet.pl, google.ro, google.rs, sme.sk, google.es, expressen.se, google.ca, nieuwsblad.be, vecernji.hr, tv2.dk, skroutz.gr, idokep.hu, emag.ro, kupujemprodajem.com, google.sk, marca.com, google.se, google.ch, bild.de, krone.at, 24sata.hr, google.dk, google.fi, origo.hu, interia.pl, olx.ro, bazos.sk, google.ie, google.co.uk, t-online.de, 24chasa.bg, ekstrabladet.dk, repubblica.it, independent.ie, dagbladet.no, google.pl, willhaben.at, blitz.bg, leboncoin.fr, ilmeteo.it, telegraaf.nl, abola.pt, gsp.ro, as.com, olx.ua, ebay.co.uk, google.com.mx, ebay-kleinanzeigen.de, slobodnadalmacija.hr.

Updating 3 rules for:
digicert.com, webex.com, indeed.com

### CODE REVIEW

File: cookie-banner-rules-list.json [hunk 1]
```diff
@@ -893,6 +893,1279 @@
...
+      "cookies": {},
+      "id": "24c6ffc2-7a11-4401-a1fe-2cec15a1a76d"
+    },
+    {
+      "click": {
+        "optIn": "button#L2AGLb",
+        "optOut": "button#W0wltc",
+        "presence": "div.spoKVd"
+      },
+      "domain": "google.at",
```
Comment (Trikolon) [inline cookie-banner-rules-list.json:1013]: For the other Google domains we use SOCS and CONSENT cookies. Are those not sufficient here?

File: cookie-banner-rules-list.json [hunk 2]
```diff
@@ -893,6 +893,1279 @@
...
+      "click": {
+        "optIn": "button.submitAll",
+        "presence": "div.submitButton"
+      },
+      "domain": "dr.dk",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "CookieConsent",
+            "value": "{stamp:%27WCJvxz/hqBZ/ELOL5sat3WmYnBbYDKFGWYlhItSfcWe2HkSYX3Fi8g==%27%2Cnecessary:true%2Cpreferences:false%2Cstatistics:false%2Cmarketing:false%2Cver:5%2Cutc:1664957292568%2Cregion:%27ro%27}"
```
Comment (Trikolon) [inline cookie-banner-rules-list.json:1107]: Is this really the opt-in cookie? It looks very similar to the opt-out one. Looks like `statistics` and `marketing` are set to `false`.

File: cookie-banner-rules-list.json [hunk 3]
```diff
@@ -893,6 +893,1279 @@
...
+      "domain": "iltalehti.fi",
+      "cookies": {},
+      "id": "23fcab4a-3b43-4d73-bb9e-57d86121141b"
+    },
+    {
+      "click": {
+        "optIn": "button#didomi-notice-agree-button",
+        "presence": "div#buttons"
+      },
+      "domain": "orange.fr",
```
Comment (Trikolon) [inline cookie-banner-rules-list.json:1227]: I don't get a banner on that site to begin with.

File: cookie-banner-rules-list.json [hunk 4]
```diff
@@ -893,6 +893,1279 @@
...
+        ]
+      },
+      "id": "e3af03b3-8990-45ef-abd5-f6da31bd31fe"
+    },
+    {
+      "click": {
+        "optIn": "button.cmp-intro_acceptAll",
+        "presence": "div.cmp-intro_options"
+      },
+      "domain": "onet.pl",
```
Comment (Trikolon) [inline cookie-banner-rules-list.json:1305]: No banner with feature disabled (with German origin IP)

File: cookie-banner-rules-list.json [hunk 5]
```diff
@@ -893,6 +893,1279 @@
...
+          {
+            "name": "FCCDCF",
+            "value": "%5Bnull%2Cnull%2Cnull%2C%5B%22CPgX9oAPgX9oAEsABCENCjCgAAAAAAAAABpYAAAOhQD2F2K2kKEkfjSUWYAQBCujIEIBUAAAAECBIAAAAUgQAgFIIAgAAlACAAAAABAQAQCAgAQABAAAoACgACAAAAAAAAAAAAQQAABAAIAAAAAAAAEAQAAAAAQAAAAAAABEhCAAQQAEAAAAAAAAAAAAAAAAAAABAAA.YAAAAAAAAAA%22%2C%221~%22%2C%22D2A36A86-FAFB-4BB0-951A-2BA964C5C018%22%5D%2Cnull%2Cnull%2C%5B%5D%5D"
+          }
+        ]
+      },
+      "id": "944c0cf0-271b-41f8-b0ca-3fa67db7af29"
+    },
+    {
+      "click": {
```
Comment (Trikolon) [inline cookie-banner-rules-list.json:1557]: Please make sure we have click rules for all google rules, including our existing google records.

**Review by Trikolon — CHANGES_REQUESTED** [review]
Thanks! 🎉
Just a couple smaller issues we should address before merging.

File: cookie-banner-rules-list.json (hunk 4 above)
Comment (Softvision-PatriciuPop) [reply cookie-banner-rules-list.json:1305]: We've looked over orange.fr and onet.pl and both have cookie banners for us on both Firefox and other browsers. Unsure why they wouldn't trigger for you in Germany.

File: cookie-banner-rules-list.json (hunk 5 above)
Comment (Softvision-PatriciuPop) [reply cookie-banner-rules-list.json:1573]: Fixed all Google click rules.

File: cookie-banner-rules-list.json (hunk 2 above)
Comment (Softvision-PatriciuPop) [reply cookie-banner-rules-list.json]: Updated the rule with the correct opt-in cookie.

File: cookie-banner-rules-list.json (hunk 1 above)
Comment (Softvision-PatriciuPop) [reply cookie-banner-rules-list.json:1029]: Updated Google rules with the correct CONSENT cookie

File: cookie-banner-rules-list.json (hunk 4 above)
Comment (Trikolon) [reply cookie-banner-rules-list.json:1321]: As long as you see banners and we handle them correctly that's ok. If no banner is shown our mechanism will simply not do anything.

**Review by Trikolon — APPROVED** [review]
r+ minus the formatting issues, you'll have to resolve those, e.g. via `npm run prettier` before this can be merged.

**Comment by Softvision-PatriciuPop** [comment]
Verified all rules work as expected on Stage, fixing broken rules/banner flashes in https://github.com/mozilla/cookie-banner-rules-list/pull/22

**Pull request merged**